src/Common/Polyfills/System/PasteArgument.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Copied from:
+// https://github.com/dotnet/runtime/blob/d2650b6ae7023a2d9d2c74c56116f1f18472ab04/src/libraries/System.Private.CoreLib/src/System/PasteArguments.cs
+// and changed from using ValueStringBuilder to StringBuilder.
+
+using System.Text;
```

Comment:
I assume this is just resurrecting the deleted file

Reply:
Nit: The filename should be PasteArgument*s*.cs